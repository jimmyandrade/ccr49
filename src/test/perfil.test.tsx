import {create} from "react-test-renderer";
import ProfilePage from "../pages/perfil";

jest.mock("next/image", () => "Image");

describe("Profile page", () => {
    it("should match snapshot", () => {
        const testRenderer = create(<ProfilePage />);
        expect(testRenderer).toMatchSnapshot();
    })
})