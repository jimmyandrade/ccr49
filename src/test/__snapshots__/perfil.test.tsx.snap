// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile page should match snapshot 1`] = `
<main
  className="MuiContainer-root MuiContainer-maxWidthLg"
>
  <h1
    className="MuiTypography-root MuiTypography-h4"
  >
    <div
      className="MuiAvatar-root MuiAvatar-circular ProfileAvatar-large-1 MuiAvatar-colorDefault"
    >
      <Image
        alt="Imagem de perfil de Dandara"
        height={77}
        src="test-file-stub"
        width={77}
      />
    </div>
     
    Dandara
  </h1>
  <p
    className="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom MuiTypography-paragraph"
  >
    <svg
      aria-hidden={true}
      className="MuiSvgIcon-root MuiSvgIcon-fontSizeSmall"
      focusable="false"
      viewBox="0 0 24 24"
    >
      <path
        d="M12 6c1.11 0 2-.9 2-2 0-.38-.1-.73-.29-1.03L12 0l-1.71 2.97c-.19.3-.29.65-.29 1.03 0 1.1.9 2 2 2zm4.6 9.99l-1.07-1.07-1.08 1.07c-1.3 1.3-3.58 1.31-4.89 0l-1.07-1.07-1.09 1.07C6.75 16.64 5.88 17 4.96 17c-.73 0-1.4-.23-1.96-.61V21c0 .55.45 1 1 1h16c.55 0 1-.45 1-1v-4.61c-.56.38-1.23.61-1.96.61-.92 0-1.79-.36-2.44-1.01zM18 9h-5V7h-2v2H6c-1.66 0-3 1.34-3 3v1.54c0 1.08.88 1.96 1.96 1.96.52 0 1.02-.2 1.38-.57l2.14-2.13 2.13 2.13c.74.74 2.03.74 2.77 0l2.14-2.13 2.13 2.13c.37.37.86.57 1.38.57 1.08 0 1.96-.88 1.96-1.96V12C21 10.34 19.66 9 18 9z"
      />
    </svg>
     
    <time
      dateTime="1997-11-20"
    >
      20/11/1997
    </time>
     (
    <data>
      24
    </data>
     anos)
  </p>
  <div
    className="MuiBox-root MuiBox-root-2"
  >
    <hr
      className="MuiDivider-root"
    />
  </div>
  <h2
    className="MuiTypography-root MuiTypography-h5"
  >
    Conquistas de 
    Dandara
  </h2>
  <ul
    className="MuiList-root MuiList-padding"
  >
    <li
      className="MuiListItem-root MuiListItem-gutters"
      disabled={false}
    >
      <div
        className="MuiListItemAvatar-root"
      >
        <div
          className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault"
        >
          <svg
            aria-hidden={true}
            className="MuiSvgIcon-root"
            focusable="false"
            viewBox="0 0 24 24"
          >
            <path
              d="M18.6 6.62c-1.44 0-2.8.56-3.77 1.53L12 10.66 10.48 12h.01L7.8 14.39c-.64.64-1.49.99-2.4.99-1.87 0-3.39-1.51-3.39-3.38S3.53 8.62 5.4 8.62c.91 0 1.76.35 2.44 1.03l1.13 1 1.51-1.34L9.22 8.2C8.2 7.18 6.84 6.62 5.4 6.62 2.42 6.62 0 9.04 0 12s2.42 5.38 5.4 5.38c1.44 0 2.8-.56 3.77-1.53l2.83-2.5.01.01L13.52 12h-.01l2.69-2.39c.64-.64 1.49-.99 2.4-.99 1.87 0 3.39 1.51 3.39 3.38s-1.52 3.38-3.39 3.38c-.9 0-1.76-.35-2.44-1.03l-1.14-1.01-1.51 1.34 1.27 1.12c1.02 1.01 2.37 1.57 3.82 1.57 2.98 0 5.4-2.41 5.4-5.38s-2.42-5.37-5.4-5.37z"
            />
          </svg>
        </div>
      </div>
      <div
        className="MuiListItemText-root MuiListItemText-multiline"
      >
        <h3
          className="MuiTypography-root MuiTypography-h6"
        >
          Sabe-Tudo
        </h3>
        <p
          className="MuiTypography-root MuiTypography-body2 MuiTypography-gutterBottom"
        >
          Ganhe 
          <data>
            500
          </data>
           pontos
        </p>
        <div
          aria-valuemax={100}
          aria-valuemin={0}
          aria-valuenow={56}
          className="MuiLinearProgress-root MuiLinearProgress-colorPrimary MuiLinearProgress-determinate"
          role="progressbar"
        >
          <div
            className="MuiLinearProgress-bar MuiLinearProgress-barColorPrimary MuiLinearProgress-bar1Determinate"
            style={
              Object {
                "transform": "translateX(-44.199999999999996%)",
              }
            }
          />
        </div>
      </div>
    </li>
  </ul>
</main>
`;
